RetailCoder.VBE/UI/Command/Refactorings/FormDesignerRefactorRenameCommand.cs

```diff
@@ -45,35 +45,39 @@ protected override void OnExecute(object parameter)
             }
         }
 
-        private Declaration GetTarget()
+        private Declaration GetTarget(QualifiedModuleName? qualifiedModuleName = null)
         {
-            var project = _vbe.ActiveVBProject;
-            var component = _vbe.SelectedVBComponent;
+            (var projectId, var component) = 
+                qualifiedModuleName.HasValue 
+                                   ? (qualifiedModuleName.Value.ProjectId, qualifiedModuleName.Value.Component)
+                                   : (Vbe.ActiveVBProject.ProjectId, Vbe.SelectedVBComponent);
```

Comment:
The selected component is always in the active project, right? So, why not just get `new QualifiedModuleName(Vbe.SelectedVBComponent)` and then work with that?

Reply:
The overload for QMN is to allow targets to be found that are not in the active project